.github/workflows/coordinator-test.yml

```diff
@@ -2,12 +2,14 @@ name: Coordinator Tests
 
 on:
   workflow_dispatch:
+  pull_request:
+    branches: [ "v1" ]
```

Comment:
why only v1?

Reply:
True, when enabling should be for `main` as well (and right now, will not matter). I added this for now to trigger CI runs for changes themselves.

NOTE: I converted this back to Draft since unfortunately CI/ITs start to fail will upgrade due to something going wrong (exception claim "too many Stargate nodes" which perhaps suggests some leak).

Reply:
Interesting. I did a separate experiment under #2378 and am seeing the same issue.